lib/workers/repository/updates/flatten.ts

```diff
@@ -13,6 +13,26 @@ import { generateBranchName } from './branch-name';
 const upper = (str: string): string =>
   str.charAt(0).toUpperCase() + str.substr(1);
 
+/**
+ * Build depNameShort
+ * #8075, #8691
+ */
+function shortenDepName(depName: string): string | undefined {
+  if (!depName) {
+    return undefined;
+  }
+
+  if (depName.startsWith('gopkg.in/')) {
+    const [pkg] = depName.replace('gopkg.in/', '').split('.');
```

Comment:
This is 100% applicable to Go Modules only, so not good to be in lib/workers

Reply:
ok, so removing the compabillity shortening. I don't think we really need that